payroll-and-expenses/sample-payroll.md

```diff
@@ -0,0 +1,220 @@
+
+We need to select Get Employee Data in predefined calculations and enter province_Name in the function field in order to get employee's province
+
+![](../assets/in-province.png)
+
+### Adding Payroll Column for Professional Tax Slap
+
+This is calculated based on employee's state and basic pay. So you need to add those two calculation columns and write the function
+to calculate pts.
+
+![](../assets/in-pts.png)
+
+### Adding Payroll Column for Employee birthday
+
+We need to select Get Employee Data in predefined calculations and enter birthday(Employees table column name) in the function field in order to get employee's province
+
+![](../assets/in-birthday.png)
+
+### Adding Payroll Column for Employee age
+
+We need to get employee's birthday as a calculation column and write a javascript function to calculate age from that.
+
+![](../assets/in-age.png)
+
+### Adding Payroll Column for Income Tax
+
+We need to use employee's age and basic pay and write a function to calculate income tax.
+
+![](../assets/in-income-tax.png)
+
+### Adding Payroll Column for ESI employee contribution
+
+
+As you have seen you can do some calculations at column level. But things such as tax which are having different percentages and multiple slabs it's better to use saved calculations. Now you should go to **Saved Calculations** tab and add a new saved calculation.
+
+You can create a saved calculation based on a Salary component group \(type\), a Salary component or an Existing payroll column. In this case we use an existing payroll column.
+
+Now click on add button on Calculation Process filed to define the actual calculation. According to our example for the full range of esi is 0.75%. So we don't need to define any ranges and can calculate the tax as follows.
+
+
+
+Now we need to create a payroll column and add this calculation method to that column.
+
+![](../assets/in-esi-employee.png)
+
+Please also play attention how we are assigning **column order** to each column.
+
+
+### Adding Payroll Column for PF employee contribution
+
+![](../assets/in-pf-employee.png)
+
+### Adding Payroll Column for Total Deductions
+
+![](../assets/in-deductions.png)
+
+### Adding Payroll Column for Other Allowances
+
+![](../assets/in-other-allowances.png)
+
+### Adding Payroll Column for ESI employer contribution
+
+![](../assets/in-esi-employer.png)
+
+### Adding Payroll Column for PF employer contribution
+
+![](../assets/in-pf-employer.png)
+
+### Adding Payroll Column for ESI Total
+
+![](../assets/in-esi-total.png)
+
+### Adding Payroll Column for PF Total
+
+![](../assets/in-pf-total.png)
+
+### Add "Net Salary" column
+
+Net Salary is Gross salary minus Total deductions. So we create Net Salary column as shown below:
+
+![](../assets/in-net-salary.png)
+
+Now we have finished defining payroll columns. When you go to Payroll columns and search "IN - " you can see all the fields in Indian payroll.
+
+## Create Payslip Template
```

Comment:
Create the Indian payslip with all related fields, and provide basic instructions. Then attach the Indian payroll export again to the ticket